app/services/xccdf_report_parser.rb

```diff
@@ -75,7 +75,13 @@ def check_for_external_reports
   end
 
   def save_missing_supported_benchmark
-    save_all_benchmark_info if SupportedSsg.versions.include?(benchmark.version)
+    save_all_benchmark_info if SupportedSsg.all.find do |ssg|
+      # There's a version mismatch in some SSGs that has to be compensated
+      if ssg.real_version.present? && ssg.real_version.to_s == benchmark.version
+        benchmark.version = ssg.version
```

Comment:
I'm not entirely sure if this is the right thing to do, but this seemed logical to me.

Reply:
I don't really understand :confused: in the corner case the `benchmark.version` will be `0.9` and the 7.2 SSG doesn't have an `upstream_version` and its `version` is the correct `0.1.25` which would never match :thinking: or am I missing something?

Reply:
You seem to understand the corner case: it's when `benchmark.version` doesn't match `upstream_version` (if available) or `version`. In that corner case, we want to overwrite the incorrect (0.9) `benchmark.version`, which is parsed from the report.